Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/Settings.java

```diff
@@ -58,6 +60,7 @@
           FILE_WIIMOTE};
 
   private HashMap<String, IniFile> mIniFiles = new HashMap<>();
+  private final HashMap<File, IniFile> mWiimoteProfileFiles = new HashMap<>();
```

Comment:
IMO it'd be better for this to be a `Map<String, IniFile>`.  It seems a bit odd to use the `File` as a key here (since only loading and saving care about the `File`, and it already assumes that `SettingsFile.getWiiProfile` returns the same `File` for the same input).  Also generally it's better to use the highest interface for the field instead of the implementation (`mIniFiles` doesn't do this, even though `java.util.Map` is already imported (and unused...)).